Source/WebCore/rendering/GridTrackSizingAlgorithm.h

```diff
@@ -120,8 +120,8 @@ class GridTrackSizingAlgorithm final {
     GridTrackSizingAlgorithm(const RenderGrid*, Grid&);
     ~GridTrackSizingAlgorithm();
 
-    void setup(GridTrackSizingDirection, unsigned numTracks, SizingOperation, std::optional<LayoutUnit> availableSpace, GridLayoutState&);
-    void run();
+    void setup(GridTrackSizingDirection, unsigned numTracks, SizingOperation, std::optional<LayoutUnit> availableSpace);
+    void run(GridLayoutState&);
```

Comment:
Some thoughts, `setup` and `run` are only each called once.

We could have `run` call `setup` and then make `setup` private.